operations/package_repository_management/Nexus_Space_Cleanup.md

```diff
@@ -6,6 +6,8 @@ issues for Nexus as well as all the services that rely on Nexus. There is no aut
 During the install of any CSM version, a large amount data is added to Nexus. If there has not been a manual cleanup of old files in Nexus,
 then there is likely to be insufficient space for the next version of CSM to be installed.
 
+**Note:** The Raspberry release has around 130 Gigabytes of space needed in Nexus
```

Comment:
Should use `***NOTE***` and I believe we do not want codewords in the docs (e.g. no Raspberry). @nrockershousen might have some feedback.

Reply:
```suggestion
**Note:** The HPE Cray EX System Software 22.07 release has around 130 Gigabytes of space needed in Nexus
```